build/distributions/CentOS/8/x86_64/DVD/ks/repodetect.sh

```diff
@@ -1,117 +1,114 @@
 #!/bin/sh
 #
-# Usage: repodetect.sh <Version> <yum_server>
+# repodetect.sh - determine kickstart repos based on your OS
+#
+# Usage: repodetect.sh VERSION [KS_SERVER] [LINUX_DIST]
+#
+#   VERSION     Major OS version number (e.g., '8', '7', '6')
```

Comment:
If this works for EL6, why wasn't the EL6 repodetect.sh replaced with a link to this file?

Reply:
It hasn't been tested from an EL6 server yet.

Reply:
We can remove the '6' support and punt on this for now if it will take too much time to test. We should drop "official" 6 support in SIMP 6.6 anyway.